.appveyor.yml

```diff
@@ -0,0 +1,74 @@
+environment:
+#  nodejs_version: "0.12.7"
+  matrix:
+    # First group is 32 bit python 3.4
+    - PYTHON: "C:\\Python34"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "32"
+
+platform: x86
+
+configuration: Release
+
+init:
+  - cmd: ECHO Processor architecture - %PROCESSOR_ARCHITECTURE%
+  - cmd: wmic OS get OSArchitecture
+
+  # As AppVeyor has multiple python install, verify which one uses by default
+  - cmd: ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%
+  - cmd: python --version
+  - cmd: python -c "import struct; print(struct.calcsize('P') * 8)"
```

Comment:
I'm not clear on the purpose of this line and others like it.  Should we test what it returns, and fail if it's the wrong thing?  Or are they just there for debugging?  (In which case, could we remove them?)